### DISCUSSION TITLE
Docs broken?

### DISCUSSION THREAD

**Discussion by cjhines** [ory/oathkeeper#1009] [Q&A]
The reference object in https://www.ory.sh/docs/oathkeeper/reference/configuration is currently empty

**Comment by vinckr** [comment]
Hey @cjhines 

Thanks for the report, will turn this into an issue in /docs.

In the meantime you can find the spec that is used to generate the reference config here: https://github.com/ory/oathkeeper/blob/master/spec/config.schema.json - not very pretty, will see to fix this in /docs...

**Reply by vinckr** [reply]
https://github.com/ory/docs/issues/994
